zospy/zpcore.py

```diff
@@ -414,11 +417,24 @@ def create_new_application(self) -> bool:
 
         self.Application = self.Connection.CreateNewApplication()
 
-        if self.Application.IsValidLicenseForAPI:
-            return True
-        else:
+        if not self.Application.IsValidLicenseForAPI:
             logger.critical("OpticStudio Licence is not valid for API, connection not established")
-            return False
+
+        assert self.Application.IsValidLicenseForAPI, "OpticStudio Licence is not valid for API, connection not established"
+        
+        return self.get_primary_system()
+ 
+    def connect_as_standalone(self) -> OpticStudioSystem:
+        """Creates a standalone Zemax Opticstudio instance.
+
+        The application will be assigned to self.Application.
```

Comment:
```suggestion
        Equal to self.create_new_application().
        
        The application will be assigned to self.Application.
```

I think it is important to acknowledge that the functions are exactly the same

Reply:
Maybe it's better to refer to `ZOS.create_new_application()`, since `self` only has a meaning inside a class